### ISSUE TITLE
Cannot find firefox browser

### ISSUE THREAD

**Issue by jaichandra** [Polymer/web-component-tester#367]
Command: 
```
polymer test -l firefox
```

Error:
```
Error: The following browsers were not found: firefox. (All installed browsers found: chrome, safari)

error:   cli runtime exception: Error: The following browsers were not found: firefox. (All installed browsers found: chrome, safari)
error:   Error: The following browsers were not found: firefox. (All installed browsers found: chrome, safari)
    at Object.<anonymous> (/Users/<user>/.nvm/versions/node/v5.7.1/lib/node_modules/polymer-cli/node_modules/wct-local/lib/browsers.js:62:19)
    at next (native)
    at fulfilled (/Users/<user>/.nvm/versions/node/v5.7.1/lib/node_modules/polymer-cli/node_modules/wct-local/lib/browsers.js:4:58)
    at process._tickCallback (node.js:406:9)
```

Firefox version: 
```
48
```

OS Version:
```
OS X Yosemite
10.10.5 (14F1909)
```
I have tried reinstalling firefox couple of times and also restarting my mac.

**Comment by joaovieira** [comment]
Same for 47. Though I believe it was fine on 46. Tested on both node v4.4.7 and v6.4.0.

**Comment by Manningham** [comment]
When running wct I get this error:
```
Installing and starting Selenium server for local browsers
----------
selenium-standalone installation starting
----------

---
selenium install:
from: https://selenium-release.storage.googleapis.com/2.53/selenium-server-standalone-2.53.1.jar
to: /usr/local/lib/node_modules/polymer-cli/node_modules/selenium-standalone/.selenium/selenium-server/2.53.1-server.jar
---
chrome install:
from: https://chromedriver.storage.googleapis.com/2.23/chromedriver_mac64.zip
to: /usr/local/lib/node_modules/polymer-cli/node_modules/selenium-standalone/.selenium/chromedriver/2.23-x64-chromedriver
---
firefox install:
from: https://github.com/mozilla/geckodriver/releases/download/v0.10.0/geckodriver-v0.10.0-mac.tar.gz
to: /usr/local/lib/node_modules/polymer-cli/node_modules/selenium-standalone/.selenium/geckodriver/0.10.0-x64-geckodriver



Error: Could not request headers from https://github.com/mozilla/geckodriver/releases/download/v0.10.0/geckodriver-v0.10.0-mac.tar.gz:

error:   cli runtime exception: Error: Could not request headers from https://github.com/mozilla/geckodriver/releases/download/v0.10.0/geckodriver-v0.10.0-mac.tar.gz:
error:   Error: Could not request headers from https://github.com/mozilla/geckodriver/releases/download/v0.10.0/geckodriver-v0.10.0-mac.tar.gz:
    at Request.<anonymous> (/usr/local/lib/node_modules/polymer-cli/node_modules/selenium-standalone/lib/install.js:392:17)
    at emitOne (events.js:90:13)
    at Request.emit (events.js:182:7)
    at Request.onRequestResponse (/usr/local/lib/node_modules/polymer-cli/node_modules/selenium-standalone/node_modules/request/request.js:1255:10)
    at emitOne (events.js:90:13)
    at ClientRequest.emit (events.js:182:7)
    at HTTPParser.parserOnIncomingClient (_http_client.js:469:21)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:103:23)
    at TLSSocket.socketOnData (_http_client.js:359:20)
    at emitOne (events.js:90:13)
```

Seems that `geckodriver-v0.10.0-mac.tar.gz` should be `geckodriver-v0.10.0-macos.tar.gz`

**Comment by panuhorsmalahti** [comment]
@Manningham I get the same problem (on Mac Os X El Capitan, Node 6.4.0) even though I my selenium-standalone has the fix applied.

**Comment by lukasoppermann** [comment]
Me too, any news on this?

**Comment by ankon** [comment]
Firefox >46 is ignored by wct-local 2.0.9+, see also https://github.com/Polymer/wct-local/issues/40 and related issues.

**Comment by jaichandra** [comment]
Only option for now seems to be to revert to Firefox 46. Even 47 didn't work.
https://github.com/SeleniumHQ/selenium/issues/2559

**Comment by btelles** [comment]
@robdodson et al. Just ran into this myself. Now that Polymer Summit is over, got a sec to take a look?

**Comment by azakus** [comment]
Firefox 47+ transitioned to the new https://github.com/mozilla/geckodriver backend, which we've had issues getting to work with WCT.
I'm going to try updating our dependency on it to the latest version and try to get 49 working with it.

**Comment by azakus** [comment]
Status Update: Looks like geckodriver v0.10 broke port number setting, but v0.11 works ok.

**Comment by azakus** [comment]
Fixed in wct-local v2.0.12

**Comment by lukasoppermann** [comment]
Now that firefox is working, how about Safari 10? https://github.com/Polymer/web-component-tester/issues/411
